docroot/themes/contrib/civictheme/civictheme_library/components/01-atoms/radio/radio.twig

```diff
@@ -5,10 +5,12 @@
  *
  * Variables:
  * - theme: [string] Theme: light, dark.
+ * - text: [string] Input field label.
```

Comment:
could you please change `text` to `content` - sorry, I wrote incorrectly in the ticket - updated now